check-cla/action.yml

```diff
@@ -3,161 +3,186 @@ name: CLA check
 description: Reacts to new PRs and check if the contributor has previously signed the conda contributor license agreement (CLA).
 inputs:
   token:
-    description: Token for commenting and labeling on contributor's PR
-    required: true
-  infrastructure_token:
-    description: Token for opening singee PR in conda/infrastructure
+    description: 'A token with ability to comment, label, and modify the commit status (`pull_request: write` and `statuses: write`)'
+    default: ${{ github.token }}
     required: true
   label:
     description: Label to apply to contributor's PR once CLA is singed
     required: true
+  cla_repo:
+    description: Upstream repository in which to create PR
+    default: conda/infrastructure
+  cla_path:
+    description: Path to the CLA signees file within the provided `cla_repo`
+    default: .clabot
+  cla_fork:
+    description: Fork of `cla_repo` in which to create branch
+    default: conda-bot/infrastructure
+  cla_token:
+    description: Token for opening singee PR in `cla_fork`
+    required: true
+  cla_author:
+    description: Git-format author/committer to use for pull request commits
+    default: Conda Bot <18747875+conda-bot@users.noreply.github.com>
 
 runs:
   using: composite
   steps:
-    - name: Get PR metadata
-      id: pr
+    # if triggered by a comment, leave a reaction
+    - name: Comment Reaction
+      uses: peter-evans/create-or-update-comment@v3.0.0
+      if: github.event_name == 'issue_comment'
+      with:
+        token: ${{ inputs.token }}
+        comment-id: ${{ github.event.comment.id }}
+        reactions: eyes
```

Comment:
Leaves a reaction on the `@conda-bot check` so we know the workflow is running (otherwise theres no indication the bot is working on it)